lib/remoteproc/remoteproc.c

```diff
@@ -258,8 +261,13 @@ int remoteproc_stop(struct remoteproc *rproc)
 		    rproc->state != RPROC_OFFLINE) {
 			if (rproc->ops->stop)
 				ret = rproc->ops->stop(rproc);
-			rproc->state = RPROC_STOPPED;
-			rproc->bitmap = 0;
+			else
+				ret = 0;
```

Comment:
Any reason to  no more consider rproc->ops->stop == NULL as an error?
if no reason, pleas remove the else case

Reply:
Yes, I made a mistake here.  Please review again, thanks!